bitwarden_license/src/Commercial.Infrastructure.EntityFramework/SecretsManager/Repositories/ProjectRepository.cs

```diff
@@ -103,4 +103,21 @@ public async Task<bool> UserHasWriteAccessToProject(Guid id, Guid userId)
 
         return await query.AnyAsync();
     }
+
+    public async Task<IEnumerable<Core.SecretsManager.Entities.Project>> ImportAsync(IEnumerable<Core.SecretsManager.Entities.Project> projects)
+    {
+        using (var scope = ServiceScopeFactory.CreateScope())
```

Comment:
Please change to using statement which removes one level of indentation.

Reply:
Updated